Jenkinsfile

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env groovy
+
+REPOSITORY = 'govuk_navigation_helpers'
+
+node {
+  def govuk = load '/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy'
+
+  try {
+    stage('Checkout') {
+      checkout scm
+    }
+
+    stage('Clean') {
+      govuk.cleanupGit()
+      govuk.mergeMasterBranch()
+    }
+
+    stage('Bundle') {
+      echo 'Bundling'
+      sh("bundle install --path ${JENKINS_HOME}/bundles/${JOB_NAME}")
+    }
+
+    stage('Linter') {
+      govuk.rubyLinter()
+    }
+
+    stage('Tests') {
+      govuk.contentSchemaDependency('${SCHEMA_GIT_COMMIT:-"deployed-to-production"}')
```

Comment:
Where is `SCHEMA_GIT_COMMIT` set? Do you know what it was used for in the Jenkins 1 build?